packages/components/src/components/div100vh-container/div100vh-container.tsx

```diff
@@ -12,6 +11,18 @@ import Div100vh from 'react-div-100vh';
 */
 /* To manually remove rvh calculation and revert to default browser calculation use is_disabled */
 /* To bypass usage of component altogether, use is_bypassed */
+
+type TDiv100vhContainerProps = {
+    id?: string;
+    children: ReactNode;
+    height_offset?: string;
+    is_bypassed?: boolean;
+    is_disabled?: boolean;
```

Comment:
```suggestion
    is_bypassed: boolean;
    is_disabled: boolean;
```
@iman-fs Instead give them a default value of `false`.